examples/tutorial_bias_advertising.ipynb

```diff
@@ -1,5 +1,12 @@
 {
  "cells": [
+   {
+      "cell_type": "markdown",
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/msdokania/AIF360/blob/master/examples/tutorial_bias_advertising.ipynb#scrollTo=iEhpr5Vxfy4g)\n"
```

Comment:
Is this `#scrollTo=iEhpr5Vxfy4g` needed?

Reply:
No this is not needed. Thank you very much. I will remove